freegs/jtor.py

```diff
@@ -41,9 +25,10 @@
 
 from numpy import clip, zeros, reshape, sqrt, pi
 import numpy as np
+import abc
 
 
-class Profile(object):
+class Profile(metaclass=abc.ABCMeta):
```

Comment:
Nice! You can inherit directly from `ABC` and avoid the metaclass stuff:

```suggestion
class Profile(abc.ABC):
```